### PR TITLE
[docs] readability and consistency edits

### PR DESCRIPTION
**Pull request by databyjp**
### Type of change:

- [x] Documentation updates (non-breaking change which updates documents)

### CODE REVIEW

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-cohere.md [hunk 1]
```diff
@@ -48,15 +50,20 @@ services:
 ...
 ```
 
-* Note: you can also use the [Weaviate configuration tool](../../installation/docker-compose.md#configurator) to create a Weaviate setup with this module.
+:::note
+* You can also use the [Weaviate configuration tool](../../installation/docker-compose.md#configurator) to create a Weaviate setup with this module.
+* Starting with `v1.11.0` the `COHERE_APIKEY` variable is now optional and you can instead provide the key at insert/query time as an HTTP header.
```
Comment (dandv) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-cohere.md:55]: * Should we mention the name of the HTTP header?
* Should we consistently format version numbers as code? The vast majority I've seen were plain text.

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-huggingface.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ The `text2vec-huggingface` module allows you to use [Hugging Face models](https:
 
 * Note: this module uses a third-party API.
 * Note: make sure to check the Inference [pricing page](https://huggingface.co/inference-api#pricing) before vectorizing large amounts of data.
-* Note: Weaviate automatically parallelizes requests to the Inference-API when using the batch endpoint, see the previous note.
+* Note: Weaviate automatically parallelizes requests to the Inference-API when using the batch endpoint.
```
Comment (dandv) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-huggingface.md:17]: "inference API"

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-huggingface.md [hunk 3]
```diff
@@ -46,8 +46,10 @@ services:
       HUGGINFACE_APIKEY: sk-foobar # request a key on huggingface.co, setting this parameter is optional, you can also provide the API key on runtime
       CLUSTER_HOSTNAME: 'node1'
 ```
-
-* Note: you can also use the [Weaviate configuration tool](/developers/weaviate/installation/docker-compose.md#configurator) to create a Weaviate setup with this module.
+:::note
+* You can also use the [Weaviate configuration tool](/developers/weaviate/installation/docker-compose.md#configurator) to create a Weaviate setup with this module.
+* Starting with `v1.11.0` the `HUGGINFACE_APIKEY` variable is now optional and you can instead provide the key at insert/query time as an HTTP header.
```
Comment (dandv) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-huggingface.md:51]: * Should we mention the name of the HTTP header?
* Should we consistently format version numbers as code? The vast majority I've seen were plain text.

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-transformers.md [hunk 4]
```diff
@@ -10,11 +10,13 @@ import Badges from '/_includes/badges.mdx';
...
+The `text2vec-transformers` module allows you to run your own inference container with a pre-trained language transformer model as a Weaviate vectorization module. Note that this is in contrast to an API-based module such as [`text2vec-openai`](./text2vec-openai.md), [`text2vec-cohere`](./text2vec-cohere.md) and [`text2vec-huggingface`](./text2vec-huggingface.md) which use an external API to vectorize your data.
 
-To use transformers with Weaviate, the `text2vec-transformers` module needs to be enabled. The models are encapsulated in Docker containers. This allows for efficient scaling and resource planning. Neural-Network-based models run most efficiently on GPU-enabled serves, yet Weaviate is CPU-optimized. This separate-container microservice setup allows you to very easily host (and scale) the model independently on GPU-enabled hardware while keeping Weaviate on cheap CPU-only hardware.
+With the `text2vec-transformers` module, you can use one of any number of a pretrained NLP module specific to your use case. This means models like `BERT`, `DilstBERT`, `RoBERTa`, `DilstilROBERTa`, etc. can be used out-of-the box with Weaviate.
```
Comment (dandv) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-transformers.md:15]: "one of any number of pretrained NLP models"?

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-transformers.md [hunk 5]
```diff
@@ -10,11 +10,13 @@ import Badges from '/_includes/badges.mdx';
...
 
-To choose your specific model, you simply need to select the correct Docker container. There is a selection of pre-built Docker images available, but you can also build your own with a simple two-line Dockerfile.
+The models are encapsulated in Docker containers. This allows for efficient scaling and resource planning. Neural-Network-based models run most efficiently on GPU-enabled serves, yet Weaviate is CPU-optimized. This separate-container microservice setup allows you to very easily host (and scale) the model independently on GPU-enabled hardware while keeping Weaviate on cheap CPU-only hardware.
```
Comment (dandv) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-transformers.md:17]: "on GPU-enabled serveRs"

**Review by dandv — CHANGES_REQUESTED** [review]
LGTM overall!

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-huggingface.md (hunk 2 above)
Comment (databyjp) [reply developers/weaviate/modules/retriever-vectorizer-modules/text2vec-huggingface.md:17]: I will remove the dash; but I think it should be capitalized as it is what their service is called. (https://huggingface.co/inference-api)

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-cohere.md (hunk 1 above)
Comment (databyjp) [reply developers/weaviate/modules/retriever-vectorizer-modules/text2vec-cohere.md:55]: 1) Good point - I've edited this bit. 
2) Personally I prefer it as code.

**Pull request merged**